blueprints/node-webkit/index.js

```diff
@@ -50,5 +51,20 @@ module.exports = {
         return writeFile(packageJsonPath, JSON.stringify(json, null, '  '));
       }
     });
+  },
+  addQUnitLogger: function(){
+    var testHelperFile = path.resolve(this.project.root, 'tests/test-helper.js');
+    var _this = this;
+    
+    return readFile(testHelperFile, { encoding: 'utf8'} ).then(function(data){
+      var code = 'require(\'' + _this.project.name() + '/tests/helpers/qunit-logger\');';
```

Comment:
Agreed. Changing to import 'vendor/node-webkit/qunit-logger' and defining that from quint-logger.js itself. still has to be imported from here as thats when Qunit will be in the global scope.